### ISSUE TITLE
menu opening icon above the first email icon in the mobile version

### ISSUE THREAD

**Issue by tigernero79** [nextcloud/mail#1958]
mail application in the mobile version from this annoying overlap of the 3 lines that make the menu open with the first readable email.

I attach screenshot

![63912980-e1f8dc00-ca2e-11e9-9b25-7bf53d05005c](https://user-images.githubusercontent.com/35575744/63923270-889ca700-ca46-11e9-8123-4217431178d1.png)

**Comment by ChristophWurst** [comment]
Yep, it's hard to star the first message then.

cc @skjnldsv because this also affects any other app that uses the app content list with a star icon

Please use the bug template for bug reports next time. This makes it easier for us to triage and understand reports. Thanks.

**Comment by skjnldsv** [comment]
Yeah, i had no idea so far, we have the same on contacts :)

**Comment by ChristophWurst** [comment]
> Yeah, i had no idea so far, we have the same on contacts :)

A problem shared is a problem halved.

Maybe we can find a better design for this. Any ideas @nextcloud/designers?

**Comment by jancborchardt** [comment]
Yep, this is a problem in lots of places, especially in the apps using the app-content-list (Mail and Contacts as you said). The menu toggle was always an afterthought so far, and we should fix that.

Here’s 3 proposals:

1. **Half menu icon:** This is how it _used to be_ in Android. Not the best solution as people don’t see the menu icon and … what should be the clickable area? No wonder why Android moved away from this.
![menu placement1](https://user-images.githubusercontent.com/925062/64024582-0510b200-cb3b-11e9-8fd0-7a0c6e1d2ea3.png)
2. **Menu + logo:** Moving everything in the header over to insert a menu icon. This is a compromise still showing the logo, but will take a lot of space out of the header for icons. And well, there’s already so much stuff in the header.
![menu placement2](https://user-images.githubusercontent.com/925062/64024581-0510b200-cb3b-11e9-82da-43ba668413aa.png)
3. **Menu, no logo:** Basically replacing the logo with the menu icon. This is of course the harshest because the logo will not be shown at all on mobile. But since the logo duplicates the primary app shortcut (in the default case Files) you could say that is ok.
![menu placement3](https://user-images.githubusercontent.com/925062/64024580-0510b200-cb3b-11e9-93fb-50f0caa061be.png)


--------------------

A middle ground we could go for is 2+3 depending on screen size. That is on tablets the logo would still be shown, but on smartphones not. Remember this view is only for the mobile web interface when you are logged in, so we kind of assume people use the mobile app, and do not necessarily need to see the logo.

Another addition to option 3: Since the menu icon is in the header, the navigation could open full-height and show the logo in the top part, just like we do in the Android app.

Any other ideas? And what do you think @nextcloud/designers?

Also cc @karlitschek @jospoortvliet since this is related to our branding.
